includes/polling/os/trango.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+$version = trim(snmp_get($device, ".1.3.6.1.4.1.5454.1.80.1.1.2.0", "-Oqv", "GIGA-PLUS-MIB"), '"');
```

Comment:
You are using numerical OIDs here but referencing the MIB, you can either change to using the named OID or drop the MIB name as it won't be used.

Same for the below.